index.html

```diff
@@ -99,7 +99,7 @@ <h2>Project Title</h2>
                         <li>CSS</li>
                         <li>JavaScript</li>
                     </ul>
-                    <a href="#"> <button class="btn" type="button">See Project</button> </a>
+                    <button id="project-one" class="get-project btn" type="button">See Project</button>
```

Comment:
- [ ] You did a great job working on the popup window! 💯 However, there's still a requirement regarding the projects section that is not yet met (point 3):

![image](https://user-images.githubusercontent.com/77069011/202529076-ef6162d6-77f7-4bbf-b956-7d3ce4267afe.png)

At the moment, your project section is statically rendered by your HTML. If you pay close attention when you click a project, the project card, and the popup window have different titles. They are working as expected, you're getting the information right, but what you have in your HTML and your projects array (JSON) differ. 🚧 

Making this section dynamically displayed based on the same projects array will ensure that you only have one source of information for both, your projects section and your popup window, making it easier to add, update or delete any projects in the future by only changing the data in your JSON file! 🤯 

Please, consider implementing this requirement! 💯 🤓 🛠️ 

💡 _You could iterate over your array of projects and build a template string (just as you did with your popup HTML) and finally append it to a reference container in your `index.html`_